lib/v2/javdb/utils.js

```diff
@@ -2,7 +2,7 @@ const got = require('@/utils/got');
 const cheerio = require('cheerio');
 const { parseDate } = require('@/utils/parse-date');
 const config = require('@/config').value;
-const allowDomain = ['javdb.com', 'javdb36.com', 'javdb007.com'];
+const allowDomain = ['javdb.com', 'javdb36.com', 'javdb521.com'];
```

Comment:
`javdb007.com` still works
```suggestion
const allowDomain = ['javdb.com', 'javdb36.com', 'javdb007.com', 'javdb521.com'];
```

Reply:
👌